input/mapping/IJE_File_Layouts_Version_2021_FHIR-2023-02-22-All-Combined.csv

```diff
@@ -828,7 +828,7 @@ D = Donation
 E = Entombment
 R = Removal from state
 O = Other
-U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[FetalRemainsDispositionMethodVS],,y,,,,DecedentDispositionMethod,
+U = Unknown",,VRDR,DecedentDispositionMethod,value,codeable,[MethodOfDispositionVS],,y,,,,DecedentDispositionMethod,
```

Comment:
I don't think this is right.

Reply:
Same as above- this is for VRDR which uses different VS